docusaurus/docs/adding-custom-environment-variables.md

```diff
@@ -10,12 +10,16 @@ Your project can consume variables declared in your environment as if they were
 default you will have `NODE_ENV` defined for you, and any other environment variables starting with
 `REACT_APP_`.
 
+> WARNING: Do not store any secrets (such as API keys) in your React app!
```

Comment:
Maybe emphasize privacy (as in `private API keys`), as certain popular API keys (such as Google Maps) are public by design and rely on things like origins to "secure" them.